src/components/dataSet-settings.js

```diff
@@ -409,7 +437,7 @@ class DataSetSettings extends React.Component {
                     api.createNamespace(
                         'ANDROID_SETTING_APP',
                         'dataSet_settings'
-                    ).catch(e => console.log(e))
+                    ).catch(e => console.error(e))
```

Comment:
I see this pattern being used a lot - `.catch(e => console.error(e))`.  It's good to catch the error so it doesn't bubble up to the top level, but this also means that things will just "not work" without any indication to the user that something went wrong.  A good pattern, in general, is to display a user-friendly error when something goes wrong.  This doesn't need to be fixed in this PR, but is something to think about for a future enhancement